### ISSUE TITLE
`install_github()` error

### ISSUE THREAD

**Issue by dajmcdon** [cmu-delphi/delphi-epidata-r#16]
I can't currently install the package. I get the following error:

```{r}
remotes::install_github("cmu-delphi/delphi-epidata-r")
```

```
The downloaded source packages are in
	‘/private/var/folders/fr/49035_g925n4f5g72mjntmth0000gn/T/RtmpvjwfKK/downloaded_packages’
✓  checking for file ‘/private/var/folders/fr/49035_g925n4f5g72mjntmth0000gn/T/RtmpvjwfKK/remotes7e0d69325f3d/cmu-delphi-delphi-epidata-r-5160ad9/DESCRIPTION’ ...
─  preparing ‘delphi.epidata’:
✓  checking DESCRIPTION meta-information ...
   Warning in grepl(e, files, perl = TRUE, ignore.case = TRUE) :
     PCRE pattern compilation error
   	'unrecognized character follows \'
   	at 'Makefile$'
   Error in grepl(e, files, perl = TRUE, ignore.case = TRUE) : 
     invalid regular expression '^\Makefile$'
   Execution halted
Error: Failed to install 'delphi.epidata' from GitHub:
  System command 'R' failed, exit status: 1, stdout & stderr were printed
```

**Comment by krivard** [comment]
Thanks for the report! This package is alpha and still has some build problems; I don't think we've even attempted to install using `remotes` yet.

**Comment by dshemetov** [comment]
Hm, unable to reproduce on my Ubuntu machine, `remotes` installs fine.

**Comment by krivard** [comment]
This may have been fixed by #14  -- would you try again?

**Comment by dajmcdon** [comment]
Yup! Works now. A got a few warnings that I'll paste here in case its useful for testing, but otherwise it's good.

```
> devtools::install_github("cmu-delphi/delphi-epidata-r")
Using github PAT from envvar GITHUB_PAT
Downloading GitHub repo cmu-delphi/delphi-epidata-r@HEAD
✓  checking for file ‘/private/var/folders/fr/49035_g925n4f5g72mjntmth0000gn/T/RtmpG4OjsG/remotes16e353aafca2d/cmu-delphi-delphi-epidata-r-067fcf8/DESCRIPTION’ ...
─  preparing ‘delphi.epidata’:
✓  checking DESCRIPTION meta-information ...
─  checking for LF line-endings in source and make files and shell scripts
─  checking for empty or unneeded directories
   Omitted ‘LazyData’ from DESCRIPTION
─  creating default NAMESPACE file
─  building ‘delphi.epidata_1.0.0.tar.gz’
   
── Attaching packages ─────────────────────────────────────── tidyverse 1.3.1 ──
✓ ggplot2 3.3.5     ✓ purrr   0.3.4
✓ tibble  3.1.6     ✓ dplyr   1.0.8
✓ tidyr   1.2.0     ✓ stringr 1.4.0
✓ readr   2.1.2     ✓ forcats 0.5.1

* installing *source* package ‘delphi.epidata’ ...
** using staged installation
** R
** byte-compile and prepare package for lazy loading
Warning: replacing previous import ‘jsonlite::unbox’ by ‘rlang::unbox’ when loading ‘delphi.epidata’
Warning: replacing previous import ‘jsonlite::flatten’ by ‘rlang::flatten’ when loading ‘delphi.epidata’
** help
No man pages found in package  ‘delphi.epidata’ 
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
Warning: replacing previous import ‘jsonlite::unbox’ by ‘rlang::unbox’ when loading ‘delphi.epidata’
Warning: replacing previous import ‘jsonlite::flatten’ by ‘rlang::flatten’ when loading ‘delphi.epidata’
** testing if installed package can be loaded from final location
Warning: replacing previous import ‘jsonlite::unbox’ by ‘rlang::unbox’ when loading ‘delphi.epidata’
Warning: replacing previous import ‘jsonlite::flatten’ by ‘rlang::flatten’ when loading ‘delphi.epidata’
** testing if installed package keeps a record of temporary installation path
* DONE (delphi.epidata)
```
